src/auto/combo_box.rs

```diff
@@ -447,6 +435,20 @@ impl<O: IsA<ComboBox> + IsA<glib::object::Object> + glib::object::ObjectExt> Com
         }
     }
 
+    fn get_property_active(&self) -> i32 {
```

Comment:
The property functions should disappear (but notify should stay), otherwise looks good to me